istio.VERSION

```diff
@@ -2,8 +2,8 @@
 # install/updateVersion.sh (see install/README.md)
 export CA_HUB="docker.io/istio"
 export CA_TAG="0.2.1"
-export MIXER_HUB="docker.io/istio"
-export MIXER_TAG="0.2.1"
-export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/fb947d4c6d0902a841e0e82ffa3bee16786be0df/artifacts/istioctl"
+export MIXER_HUB="gcr.io/istio-testing"
+export MIXER_TAG="2c8e76790aa1d569b739454ca2dba5e0b031bd25"
+export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/b5d710d658d3bfffa06506f312725d96a8737703/artifacts/istioctl"
```

Comment:
**merge error**, please leave fb947d4c6d0902a841e0e82ffa3bee16786be0df
(and again, I think you should use latest mixer (49ad083ba35943880f57a16a475c50b1e037dde8) given the big change and need to debug the tests anyway)

Reply:
fixed it, PTAL.